reporting-app/app/models/ex_parte_activity.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+# ExParteActivity stores hours data submitted via API or batch upload.
+#
+# These are "ex parte" (automated/external) hours as opposed to manually
+# reported hours from ActivityReportApplicationForm. Ex parte hours are
+# auto-verified and don't require staff review.
+#
+# @note The `outside_period` flag should be set by the service layer
+#   (ExParteActivityService) when creating entries, not by callbacks.
+#
+class ExParteActivity < ApplicationRecord
+  ALLOWED_CATEGORIES = %w[employment community_service education].freeze
+
+  SOURCE_TYPE_API = "api"
+  SOURCE_TYPE_BATCH = "batch_upload"
+  ALLOWED_SOURCE_TYPES = [ SOURCE_TYPE_API, SOURCE_TYPE_BATCH ].freeze
+
+  MAX_HOURS = 744
+
+  # --- Associations ---
+
+  belongs_to :certification, optional: true
```

Comment:
Follow [the data modeling guidelines](https://github.com/navapbc/strata-sdk-rails/blob/68483d71a15444796864ae50ff7073812c336247/docs/contributing/data-modeling-guidelines.md) and avoid adding associations between aggregates